styles/index.css

```diff
@@ -0,0 +1,192 @@
+@import "//netdna.bootstrapcdn.com/font-awesome/3.0/css/font-awesome.css";
+
+.navbar-container {
+  height: 540px;
+  width: 100%;
+  background-image: url("../img/main-img.png");
+  background-size: cover;
+  margin-top: -2%;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  width: 700px;
+  margin: 50px auto auto auto;
+  text-align: center;
+  background:
+    linear-gradient(
+      to bottom,
+      rgba(125, 126, 125, 1) 0%,
+      rgb(71, 70, 70) 47%,
+      rgba(142, 142, 142, 1) 92%
+    );
+  height: 36px;
+}
+
+ul {
+  list-style: none;
+}
+
+.navbar div {
+  border-right: 2px ridge rgb(155, 153, 153);
+  width: 102px;
+  height: 36px;
+  text-align: center;
+}
+
+.navbar div a {
+  color: white;
+  text-decoration: none;
+  margin-top: 10px;
+  display: inline-block;
+  font-size: 15px;
+}
+
+.nav-links {
+  margin-top: 10%;
+  background-image: url("http://images.apple.com/global/nav/images/globalnav_text.svg");
+}
+
+section {
+  text-align: center;
+  color: white;
+}
+
+section h1 {
+  font-size: 44px;
+  line-height: 36px;
+  font-weight: 500;
+}
+
+section p {
+  font-size: 27px;
+  color: white;
+  font-weight: 100;
+  line-height: 38px;
+  font-style: normal;
+}
+
+footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 10px;
+}
+
+.footer-container1 {
+  display: flex;
+  flex-direction: column;
+  font-size: 10.0008px;
+}
+
+.footer-container1 div p a {
+  color: #08c;
+  text-decoration: none;
+}
+
+.footer-container2 {
+  margin-top: -1%;
+}
+
+li {
+  border-right: 1px solid #ccc;
+  color: #08c;
+  padding-right: 8px;
+  padding-left: 8px;
+}
+
+ul li {
+  display: inline;
+  font-size: 10.0008px;
+}
+
+li:last-child {
+  border-right: none;
+}
+
+input[type="text"] {
+  width: 100%;
+  height: 19px;
+  box-sizing: border-box;
+  border-radius: 12px;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  background: transparent;
+  margin-top: 10px;
+}
+
+input[type="text"]::-webkit-input-placeholder {
+  font-weight: normal;
+  overflow: visible;
+  vertical-align: top;
+  display: inline-block !important;
```

Comment:
Remove `!important` from your CSS file, it's a [bad practice](https://speckyboy.com/good-bad-css-practices). Use more specific selectors instead.

Reply:
Done